experimental-examples/tina-cloud-starter-iframe/.tina/config.tsx

```diff
@@ -29,6 +29,14 @@ const config = defineStaticConfig({
     publicFolder: "public", // The public asset folder for your framework
     outputFolder: "admin", // within the public folder
   },
+  cmsCallback: (cms) => {
+    /**
+     * Enables experimental branch switcher
+     */
+    cms.flags.set("branch-switcher", true);
```

Comment:
We can leave this out if we want, the branch switcher now can be toggled from the banner 
<img width="414" alt="Screen Shot 2022-12-15 at 11 11 04 AM" src="https://user-images.githubusercontent.com/3323181/207896423-968d6ba7-1dae-4216-9d97-f618cfdb10e9.png">
Probably doesn't hurt to keep it and see which UI people gravitate to though